api/src/main/resources/marquez/db/migration/V62__run_state_indices.sql

```diff
@@ -0,0 +1,4 @@
+create index if not exists run_states_uuid_index on run_states(run_uuid);
+create index if not exists parent_run_uuid_index on runs(parent_run_uuid);
+create index if not exists start_run_state_uuid_index on runs(start_run_state_uuid);
+create index if not exists end_run_state_uuid_index on runs(end_run_state_uuid);
```

Comment:
Minor: I would name the migration `V62__index_run_states_and_parent_run.sql`

Reply:
Looks good, I'll make the change.

Reply:
Minor: I would also make sure to prefix the runs table indices with the table name `runs_*`